### DISCUSSION TITLE
Help Needed！Some troubles about  .strm

### DISCUSSION THREAD

**Discussion by AkimioJR** [jellyfin/jellyfin#11448] [Q&A]
Hi, I've been having some trouble with Jellyfin recently that I haven't been able to figure out, so I'm reaching out here for help.

I read in the Jellyfin documentation that it supports files with the ".strm" extension. I put an HTTPS URL link into a .strm file (opening this URL in a browser downloads an MKV video file directly). When I play this .strm file in Jellyfin, I found that the client tries to access the URL address in the .strm file. If the access is successful, the video file is obtained directly from the URL, and all traffic does not go through the Jellyfin server, unless the client cannot access the URL in the .strm file normally or the client requests transcoding.

However, after I added multiple .strm files, I found that the client no longer tries to access the URL, but instead the server directly streams the media. This takes up Jellyfin server bandwidth and wastes resources, which I don't want. What should I do?

**Comment by gagzzz** [comment]
Hello, i'd like to reopen this issue. Has it been thought of? Thank you very much, it's a really nice feature to allows one to use any remote storage backend, or share library without having to copy the storage and be able to keep a different list of media with overlapping content.

Thanks jellyfin ♥

**Comment by huplo-glitch** [comment]
+1 as I want to bypass Jellyfin transcoding which is resulting in unplayable streams across my devices. However, if I copy over the STRM file which points outside of Jellyfin and open it, it works flawlessly.
